src/utilityFunctions.js

```diff
@@ -0,0 +1,36 @@
+export const getPriceRangedData = (data, priceRange) => {
+    const [startValue, endValue] = priceRange.toString().split('-')
+    if (priceRange !== 0) {
+        if (startValue !== undefined && endValue !== undefined) {
+            return data.filter(item => item.price >= parseInt(startValue) && item.price < parseInt(endValue))
+        } else {
+            return data.filter(item => item.price > parseInt(startValue))
+        }
+    }
+    return data;
+}
+
+export const getSortedData = (data, sortType) => {
+    if (sortType === "low-to-high") {
+        return data.sort((a, b) => a['price'] - b['price'])
+    } else if (sortType === "high-to-low") {
+        return data.sort((a, b) => b['price'] - a['price'])
+    }
```

Comment:
Here you are using sort function and sort function mutates the original array. So my suggestion is that you can use spread operator while sorting.